### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by why-a-en**
#Branch changes

- Added a status checker function that checks if the task is completed or not together with the event listener
- Implemented a function that clears all completed tasks
- Stored updated array items and display them when the page reloads

### CODE REVIEW

File: src/modules/mainFunction.js [hunk 1]
```diff
@@ -64,26 +66,58 @@ class TaskList {
...
           }
           this.display();
+          this.checkStatus();
           this.saveTasks();
         });
       }
     });
   }
 
+  checkStatus = () => {
```
Comment (BenMukebo) [inline src/modules/mainFunction.js]: - [ ] It's one of the project requirements to create a new JavaScript file that will contain methods related to the status updates (completed: true/false). Kindly create a new file for the checkStatus method and imported it as a module. ✍🏻 
- [ ] Make sure to add an  (change) event listener to the checkbox.

File: src/modules/mainFunction.js [hunk 2]
```diff
@@ -64,26 +66,58 @@ class TaskList {
...
+        if (checkbox.checked) {
+          infoTask.style.textDecoration = 'line-through';
+        } else {
+          infoTask.style.textDecoration = 'none';
+        }
+      });
+    });
+  }
+
+  deleteCompletedTasks() {
```
Comment (BenMukebo) [inline src/modules/mainFunction.js:94]: - [x] According to the project requirements, The updated array of items must be stored in local storage, so the user gets the correct list values after the page reloads. However, all the checked books (completed: true) get deleted from the local storage all the time I reload the browser. Kindly prevent this 👍🏻 Here is the video screen 👇🏻 

[Screencast from 04-01-2023 11:24:57 AM.webm](https://user-images.githubusercontent.com/79058364/229277880-b45173ac-fba8-44bf-a803-c19e55cc47e4.webm)

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @anbehindY,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**